src/components/select/select.component.spec.ts

```diff
@@ -943,3 +943,81 @@ describe('Select with recent options', () => {
     });
 
 });
+
+@Component({
+    selector: 'app-select-null-value-test',
+    template: `
+        <form [formGroup]="form">
+            <ux-select
+                formControlName="select"
+                [disabled]="disabled"
+                [required]="true"
+                [options]="options">
+            </ux-select>
+        </form>
+
+        <span class="info">
+            <span>select value: <span id="select-value">{{ form.get('select')?.value }}</span></span><br>
+            <span>select errors: <span id="select-errors">{{ (form.get('select')?.errors | json) }}</span></span>
+        </span>
```

Comment:
This is a unit test, so you don't need to render and and extract values from the DOM to test.